locale/de_DE/LC_MESSAGES/django.po

```diff
@@ -4150,10 +4150,10 @@ msgid ""
 "and follow the invitation link again. If you decline the invitation the link "
 "is not valid anymore."
 msgstr ""
-"Wenn Sie akzeptieren, treten Sie dem Projekt mit Ihrem Nutzernamen "
-"\"%(user)s\" bei. Wenn Sie dem Projekt mit einem anderen Profil "
-"beitreten möchten, melden Sie sich ab und folgen Sie dem Einladungslink
```

Comment:
there was a missing `"` here which may been why it wasn't picking up a4 translations?`anyway they all seem tot be there now